tools/pd-ctl/pdctl/command/config_command.go

```diff
@@ -374,6 +375,12 @@ func postConfigDataWithPath(cmd *cobra.Command, key, value, path string) error {
 		val = value
 	}
 	data[key] = val
+	if key == "max-replicas" {
+		replica, err := strconv.ParseInt(value, 10, 64)
+		if err == nil && replica < sc.DefaultMaxReplicas {
+			cmd.Println("Setting max-replica to less than 3 may be a mistake and carries high risk. Please confirm the setting.")
```

Comment:
I think the original issue wants an interactive way to confirm the operation instead of just printing a log.

Reply:
After the discussion with @niubell , we will not add an interactive way in pd-ctl for now.

Reply:
How about not using 3?